client/src/components/ElementEditor/Header.js

```diff
@@ -16,6 +17,19 @@ class Header extends Component {
     };
   }
 
+  componentDidUpdate() {
+    if (this.state.tooltipOpen && this.props.disableTooltip) {
```

Comment:
Perhaps we could destructure these props and state attributes for consistency's sake?!